pkg/networkutils/network.go

```diff
@@ -247,6 +247,16 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 		}
 	}
 
+	if n.mtu != maximumMTU {
```

Comment:
Why guard the `LinkSetMTU()` call with this conditional? Why not just always set the MTU for the link? This conditional assumes that the link will have its default MTU set to jumbo frames, but if that changes in the future, this conditional will introduce a bug, right?

Reply:
Thanks, good catch.